packages/foundation/src/elements/navigation-drawer.css

```
@@ -0,0 +1,81 @@
+.hap-navigation-drawer {
+  display: flex;
+
+  flex-direction: column;
+  min-width: 360px;
+  width: fit-content;
+  justify-content: flex-end;
+  align-items: stretch;
+
+  gap: 0;
+  padding: var(--hap-spacing-md);
+
+  background: var(--hap-color-surface-primary-default);
+  border: none;
+
+  menu {
+    display: flex;
+    flex-direction: column;
+    align-items: stretch;
+
+    margin: 0;
+    padding: 0;
+    list-style: none;
+
+    li {
+      margin: 0;
+      padding: 0;
+    }
+  }
```

Kleiner Vorschlag, der weniger Code benötigen würde. Kannst verwenden oder ignorieren:
```suggestion
  menu {
    display: contents;
    list-style: none;
  }
```